tests/test_playbooks/convert2rhel.yml

```diff
@@ -23,8 +23,10 @@
       vars:
         foreman_organization: "Test Organization"
         foreman_validate_certs: false
+        foreman_manifest_path: "~/manifest.zip"
```

Comment:
due to `foreman_convert2rhel_manage_subscription: false` on the next line, this has no effect *plus* the path doesn't exist in CI and would fail then.